simulator/README.md

```diff
@@ -0,0 +1,23 @@
+# Modbus Simulator
+Modbus Simulator is used to create mock Modbus device for testing purpose. 
+
+## Build docker image
+The docker image built with git branch https://github.com/edgexfoundry/device-modbus-go/tree/edgex-modbus-simulator
+```
+git clone -b edgex-modbus-simulator git@github.com:edgexfoundry/device-modbus-go.git
+docker build -t modbus-simulator .
+```
+
+## Usage
+
+1. Create a default simulator with port 1502
+    ```
+    docker run --rm -p 1502:1502 modbus-simulator
+    ```
+
+2. create simulators for scalability test 
+    ```
+    docker run --rm -d --network host --name modbus-simulator  \
+        -e SIMULATOR_NUMBER=1000 -e STARTING_PORT=10000 modbus-simulator
```

Comment:
modify the example to the real available port number

Reply:
I tested the port and it work for me.